### PR TITLE
Code Updates to Algorithm Plugins, Analytic Schema and Analytic View 

### PR DESCRIPTION
**Pull request by Delphinus8821**
### Prerequisites

- [x] Reviewed the [checklist](CHECKLIST.md)

- [ ] Reviewed feedback from the "Sonar Cloud" bot. Note that you have to wait
    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change
Updated Core Algorithm Plugins, Core Analytic Schema and Core Analytic View with code features from the new Java versions and some general cleanup. 

### Why Should This Be In Core?
Updates to Core code.

### Applicable Issues
#1987

### CODE REVIEW

File: CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/compatibility/AnalyticSchemaV6UpdateProvider.java [hunk 1]
```diff
@@ -62,21 +62,10 @@ protected void schemaUpdate(final StoreGraph graph) {
...
-                        case "IPv6 Address":
-                            graph.setObjectValue(typeAttributeId, vertexId, AnalyticConcept.VertexType.IPV6);
-                            break;
-                        case "Email":
-                            graph.setObjectValue(typeAttributeId, vertexId, AnalyticConcept.VertexType.EMAIL_ADDRESS);
-                            break;
-                        case "MGRS":
-                            graph.setObjectValue(typeAttributeId, vertexId, AnalyticConcept.VertexType.MGRS);
-                            break;
-                        default:
```
Comment (antares1470) [inline CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/compatibility/AnalyticSchemaV6UpdateProvider.java:77]: same here

File: CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/AnalyticViewController.java [hunk 2]
```diff
@@ -341,9 +341,7 @@ public void selectOnInternalVisualisations(final GraphElementType elementType, f
                                 selected.add(transactionId);
                             }
                         }
-                        break;
-                    default:
```
Comment (antares1470) [inline CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/AnalyticViewController.java:345]: same here

File: CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/analytics/FactAnalyticPlugin.java [hunk 3]
```diff
@@ -99,16 +99,14 @@ protected final void computeResultsFromGraph(final GraphReadMethods graph, final
...
                     graphElementCount = graph.getVertexCount();
                     identifierAttributeId = VisualConcept.VertexAttribute.IDENTIFIER.get(graph);
-                    break;
-                case TRANSACTION:
+                }
+                case TRANSACTION -> {
                     graphElementCount = graph.getTransactionCount();
                     identifierAttributeId = VisualConcept.TransactionAttribute.IDENTIFIER.get(graph);
-                    break;
-                default:
```
Comment (antares1470) [inline CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/analytics/FactAnalyticPlugin.java:110]: same here

File: CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/analytics/ScoreAnalyticPlugin.java [hunk 4]
```diff
@@ -99,16 +99,14 @@ protected final void computeResultsFromGraph(final GraphReadMethods graph, final
...
                     graphElementCount = graph.getVertexCount();
                     identifierAttributeId = VisualConcept.VertexAttribute.IDENTIFIER.get(graph);
-                    break;
-                case TRANSACTION:
+                }
+                case TRANSACTION -> {
                     graphElementCount = graph.getTransactionCount();
                     identifierAttributeId = VisualConcept.TransactionAttribute.IDENTIFIER.get(graph);
-                    break;
-                default:
```
Comment (antares1470) [inline CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/analytics/ScoreAnalyticPlugin.java:110]: same here

**Review by antares1470 — APPROVED** [review]

**Review by Quasar985 — APPROVED** [review]

**Pull request merged**